Makefile

```diff
@@ -1,47 +1,63 @@
-ifndef GOPATH
-$(error No GOPATH set)
-endif
+SHORT_NAME := helm
+DEIS_REGISTRY ?= ${DEV_REGISTRY}
+IMAGE_PREFIX ?= helm
```

Comment:
`DEIS_REGISTRY` and `IMAGE_PREFIX` are not needed.  These are only relevant when a build is producing images that must be push to a Docker registry.  That's not applicable here since this build only produces binaries.